core/application-crypto/src/lib.rs

```diff
@@ -53,9 +53,9 @@ pub use traits::*;
 #[macro_export]
 macro_rules! app_crypto {
```

Comment:
What I wanted from you:
```
#[macro_export]
#[cfg(feature = "full_crypto")]
macro_rules! app_crypto {
    // Call 2 macros, one does the public and signature implementation and the other one does the Pair impl
}

#[macro_export]
#[cfg(not(feature = "full_crypto"))]
macro_rules! app_crypto {
    // Just call the macro that does the public and signature implementation
}
```

Reply:
Ok, I'll take care of this

Reply:
done, but I had to further split macros in order to get rid of all `full_crypto` compiler switches within macros